src/components/Recording.jsx

```diff
@@ -106,26 +102,32 @@ export default class Recording extends Component {
         };
 
         recognition.onerror = (event) => {
+            const errorLink = '//support.google.com/chrome/bin/answer.py?hl=en&amp;answer=1407892';
+            const chromeLink = 'chrome://settings/contentExceptions#media-stream-mic';
```

Comment:
Alright, switch `chromeLink` to be "//support.google.com/chrome/answer/2693767?hl=en"